api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java

```diff
@@ -112,29 +123,32 @@ public void testGetAllAppsAndIsSortedByOrder() throws Exception {
 
     @Test
     public void testGetAllEnabledAppsShouldIgnoreAppsToggledOffInFeatureTogglesFile() throws Exception {
-        List<AppDescriptor> allApps = appFrameworkService.getAllEnabledApps();
+    	AppFrameworkServiceImpl service = new AppFrameworkServiceImpl(null, allAppDescriptors, allFreeStandingExtensions, allComponentsState, null, featureToggles, appFrameworkConfig, null);
+        List<AppDescriptor> allApps = service.getAllEnabledApps();
 
         assertEquals(1, allApps.size());
         assertEquals("app1", allApps.get(0).getId());
     }
 
     @Test
     public void testGetAllEnabledAppsShouldCorrectlyHandleNegatedFeatureToggles() throws Exception {
+    	
+    	AppFrameworkServiceImpl service = new AppFrameworkServiceImpl(null, allAppDescriptors, allFreeStandingExtensions, allComponentsState, null, featureToggles, appFrameworkConfig, null);
```

Comment:
Since you are instantiating the service is so many places, how about doing it at once in the set up method?

Reply:
@dkayiwa done.